Assets/Scripts/Framework/Movement.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections;
+using FrameWork.Extensions;
+using UnityEngine;
+namespace Framework
```

Comment:
```suggestion
using UnityEngine;

namespace Framework
```
Ik vind zo mooier :D